### ISSUE TITLE
[Questions] Ask about the use of eslint'core rules

### ISSUE THREAD

**Issue by IceApriler** [typescript-eslint/typescript-eslint#840]
**Repro**

```JSON
{
  "extends": [
    "standard",
    "plugin:@typescript-eslint/recommended"
  ],
  "rules": {
      "space-before-blocks": "error",
      "space-before-function-paren": ["error"],
      "object-curly-spacing": ["error", "always"],
      "keyword-spacing": "error"
  }
}
```

```TS
// your repro code case
```

**Expected Result**
1. Does the @typescript-eslint/eslint-plugin support all of eslint's core rules, such as the above?
2. Can `eslint-config-standard` be Used for extends?

**Actual Result**
The rule did not take effect.

**Additional Info**

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `1.13.0` |
| `@typescript-eslint/parser`        | `1.13.0` |
| `TypeScript`                       | `3.5.3` |
| `ESLint`                           | `6.1.0` |
| `node`                             | `10.16.0` |
| `npm`                              | `6.9.0` |

**Comment by bradzacher** [comment]
A number of these questions are answered within our readme: https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#readme

The rules we have implemented in this plugin are listed in the readme above.

You can extend whatever you like. However, you will have to check the configs you're extending, as there are a number of rules that do not work well / at all with typescript syntax.

You will have to manually add local config so that you use our version of these rules

https://github.com/typescript-eslint/typescript-eslint/blob/428567d7cc0985b1da754f092289212df3fe1bda/packages/eslint-plugin/tools/generate-configs.ts#L25-L36

---

Closing for housekeeping as it's not an open issue, feel free to continue commenting.
